pytensor/tensor/extra_ops.py

```diff
@@ -1194,23 +1194,22 @@ def __init__(
         self.return_index = return_index
         self.return_inverse = return_inverse
         self.return_counts = return_counts
+        if axis is not None and axis < 0:
+            raise ValueError("Axis cannot be negative.")
```

Comment:
Why remove this possibility?

Reply:
Because it simplifies the logic in the Op. The helper users use `pt.unique` handles the negative axis and passes a positive one to the Op. Users don't really create Ops themselves